freud/density.pyx

```diff
@@ -571,38 +568,34 @@ cdef class GaussianDensity(Compute):
 
     @Compute._computed_property()
     def gaussian_density(self):
-        cdef unsigned int width_x = self.thisptr.getWidthX()
-        cdef unsigned int width_y = self.thisptr.getWidthY()
-        cdef unsigned int width_z = self.thisptr.getWidthZ()
-        cdef unsigned int array_size = width_x * width_y
+        cdef vec3[uint] width = self.thisptr.getWidth()
+        cdef unsigned int array_size = width.x * width.y
```

Comment:
Can you flip the box declaration with the `array_size` declaration so that the width math stays together? Or:

`cdef unsigned int array_size = width.x * width.y * (1 if box.is2D() else width.z)`